src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -410,6 +410,8 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                             source: $source,
                             image: $runtime['image'],
                             version: $version,
+                            cpus: $function->getAttribute('cpus', 1),
+                            memory: $function->getAttribute('memory', 128),
```

Comment:
Execution specs and build specs should remain different. We should either hard-code build specs (very high), or make it configurable separatelly.

Reply:
I hard-coded 2 cpu cores and 1024 memory, that should be enough right?